File: Java Programing Basics/While-Loop - Lab/src/Graduation.java
import java.util.Scanner;

public class Graduation {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        String nameStudent = scanner.nextLine();
        double middleExcellent = 0;
        int levelOfClass = 1;

        while (levelOfClass <= 12){
            double grade = Double.parseDouble(scanner.nextLine());
            if(grade >= 4.0){
                levelOfClass++;
                middleExcellent += grade;
            }
        }
        double averagegrade = middleExcellent / 12;
        System.out.printf("%s graduated. Average grade: %.2f", nameStudent, averagegrade );
    }
}

File: Python/While_Loop/Graduation.py
student_name = input()
grades_count = 1
total = 0

while grades_count <= 12:
    grade = float(input())

    if grade >= 4:
        total += grade
        grades_count += 1

average_grade = total / 12


print(f"{student_name} graduated. Average grade: {average_grade:.2f}")